frontend/src/scenes/appScenes.ts

```diff
@@ -39,6 +39,7 @@ export const appScenes: Record<Scene, () => any> = {
     [Scene.EarlyAccessFeature]: () => import('./early-access-features/EarlyAccessFeature'),
     [Scene.Surveys]: () => import('./surveys/Surveys'),
     [Scene.Survey]: () => import('./surveys/Survey'),
+    [Scene.SurveyTemplates]: () => import('./surveys/SurveyTemplates'),
```

Comment:
It actually still routes to `survey_templates` which is its own scene, I was thinking a bit about this on whether to just incorporate it directly onto the surveys components or not but I think a separation of routes is the better option. That way users can always be directed to the templates page instead of it being a "state"